### PR TITLE
Fix ingestion from other schema

### PR DESCRIPTION
**Pull request by daniellizardons**
[Asana - Postgres data source fails to ingest from other schemas](https://app.asana.com/0/1107836860704912/1131720529130989/f)

## Description
- Include `schema` name when querying for table metadata like indexes/columns.

### Test
- Reproduced and tested against this DB
image

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="panoply_postgres",
-    version="2.3.3",
+    version="2.3.4",
```
Comment (kensaggy) [inline setup.py]: Is this really a backward compatible patch? I mean is it simply a matter of listing tables from other schemas? or will this affect the `__tableName` as well? What if we have the same table name in multiple schemas?
e.g. `schema_a.users` and `schema_b.users`

@alonbrody Thoughts about how we should distribute the data into tables in the public schema in panoply?

Comment (daniellizardons) [reply setup.py]: @kensaggy You're right. This scenario was not considered in this PR.
> What if we have the same table name in multiple schemas?
> e.g. schema_a.users and schema_b.users

Comment (Offir) [reply setup.py]: @daniellizardons can you check this scenario?

Comment (daniellizardons) [reply setup.py]: @Offir I ran against with `source-postgres` with version `v2.2.2`, the version before my previous PR, **and** with my latest changes. The tables generated are the same.

**Table with same name and structure**
`public`.`same_name_structure`   

| id  | name |                                                           
| ------------- | ------------- |
| 1  | a  |
| 2  | b  |
| 3  | c  |

`newschema`.`same_name_structure`

| id  | name |
| ------------- | ------------- |
| 4  | d  |
| 5  | e  |
| 6  | f   |

**Generated table in Panoply**

| id  | name | __tablename | __schemaname |
| ------------- | ------------- | ------------- | ------------- |
| 1  | a  | same_name_structure | public |
| 2  | b  | same_name_structure | public |
| 3  | c  | same_name_structure | public |
| 4  | d  | same_name_structure | newschema |
| 5  | e  | same_name_structure | newschema |
| 6  | f   | same_name_structure | newschema |

------------------------------------------------------------------------
**Table with same name, data and structure**
`public`.`same_name_data_structure`   

| id  | name |                                                           
| ------------- | ------------- |
| 1  | a  |
| 2  | b  |
| 3  | c  |

`newschema`.`same_name_data_structure`

| id  | name |
| ------------- | ------------- |
| 1  | a  |
| 2  | b  |
| 3  | c  |

**Generated table in Panoply**

| id  | name | __tablename | __schemaname |
| ------------- | ------------- | ------------- | ------------- |
| 1  | a  | same_name_structure | newschema |
| 2  | b  | same_name_structure | newschema |
| 3  | c  | same_name_structure | newschema |


The tables generated are the same before and after my changes.

**Pull request merged**